extras/havpe-relay/init.py

```
@@ -0,0 +1,425 @@
+            self.console.print(
+                f"[blue][INFO][/blue] Backed up existing .env file to {backup_path}"
+            )
+
+    def setup_backend_urls(self):
+        """Configure backend URL and WebSocket URL"""
+        self.print_section("Backend Connection")
+        self.console.print("Configure how the relay connects to the Chronicle backend")
+        self.console.print()
+
+        default_http = "http://host.docker.internal:8000"
+        default_ws = "ws://host.docker.internal:8000"
```

Ruff (0.15.2)

[error] Local variable `default_ws` is assigned to but never used

Remove assignment to unused variable `default_ws`

(F841)